core/api/service/state/state_jrpc_processor.cpp

```diff
@@ -34,9 +34,15 @@ namespace kagome::api::state {
     server_->registerHandler("state_getStorage",
                              Handler<request::GetStorage>(api_));
 
+    server_->registerHandler("state_getStorageAt",
+                             Handler<request::GetStorage>(api_));
+
     server_->registerHandler("state_getRuntimeVersion",
                              Handler<request::GetRuntimeVersion>(api_));
 
+    server_->registerHandler("chain_getRuntimeVersion",
```

Comment:
I guess it should be
```suggestion
    server_->registerHandler("state_getRuntimeVersion",
```

Reply:
Required for compatibility with python's substrate library

Reply:
Please, comment it in code to avoid question in future